cli/config_file.rs

```diff
@@ -340,6 +363,16 @@ impl ConfigFile {
       Ok((json!({}), None))
     }
   }
+
+  pub fn as_lint_config(&self) -> Result<Option<LintConfig>, AnyError> {
+    if let Some(config) = self.json.lint.clone() {
+      let lint_config: LintConfig = serde_json::from_value(config)
+        .context("Failed to parse \"lint\" configuration")?;
+      Ok(Some(lint_config))
+    } else {
+      Ok(None)
+    }
+  }
```

Comment:
Rust API guidelines say that APIs that are expensive to call like this should be prefixed with `to_` rather than `as_`. 
https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv
This API is not exposed externally (since deno is a binary crate) so it's not very important to follow naming convention though.

Reply:
Addressed